CRAYSAT-1495: Document limitations to `sat swap` blade replacement procedure (#2181)
Cray-HPE/docs-csm: operations/node_management/Replace_a_Compute_Blade.md

       -X PUT -d @newHSN.json
       ```

19. Optional: If necessary, reload DVS on NCNs.

>     DVS should be reloaded if it is running over the NMN. It is recommended that DVS be run over the HSN for simplified management and significant performance benefits.

Reviewer: ```suggestion
    Reload DVS if it is running over the NMN. The recommendation is to run DVS over the HSN for simplified management and significant performance benefits.
```

>     See *HPE Cray Operating System Administration Guide: CSM on HPE Cray EX Systems (S-8024)* for more information.

Reviewer: ```suggestion
    For more information, see *HPE Cray Operating System Administration Guide: CSM on HPE Cray EX Systems (S-8024)*.
```
